Directory.Build.props

```diff
@@ -47,4 +50,16 @@
 
   <!-- Build with the hybrid CRT (Universal CRT + Static VS CRT (for what little the Universal CRT doesn't cover) -->
   <Import Project="$(MSBuildThisFileDirectory)HybridCRT.props" />
+
+  <!-- Load the test certificate's password (so we do it once for reuse across projects) -->
+  <PropertyGroup Condition="'$(RepoTestCertificatePFX)' == ''">
+    <RepoTestCertificatePFX>$(RepoRoot)\.user\winappsdk.certificate.test.pfx</RepoTestCertificatePFX>
```

Comment:
I believe the dev\MRTCore\mrt\Core\src\MRM.vcxproj is pulling this property file and because this project is being built in a different job. It started failing.

Reply:
DevCheck.ps1 will need to run in build\build-mrt.yml to fix this. build-mrt.yml is run in a different job than build\AzurePipelinesTemplates\WindowsAppSDK-BuildProject-Steps.yml
or
or this property be moved so it's not in directory.build.props